app/controllers/aspects_controller.rb

```diff
@@ -51,22 +49,20 @@ def destroy
   end
 
   def show
-    if @aspect = current_user.aspects.where(:id => params[:id]).first
-      redirect_to aspects_path('a_ids[]' => @aspect.id)
+    if aspect
+      redirect_to aspects_path('a_ids[]' => aspect.id)
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.